src/syslog_error_h.erl

```diff
@@ -141,9 +141,9 @@ handle_msg({warning_msg, _, {Pid, Fmt, Args}}, State) ->
 handle_msg({warning_report, _, {Pid, Type, Report}}, State) ->
     log_report(warning, Pid, Type, Report, State);
 handle_msg({info_msg, _, {Pid, Fmt, Args}}, State) ->
-    log_msg(notice, Pid, Fmt, Args, State);
+    log_msg(informational, Pid, Fmt, Args, State);
 handle_msg({info_report, _, {Pid, Type, Report}}, State) ->
```

Comment:
Are you sure that you need this (progress reports can completely be disabled using `{no_progress, true}`)?

I'm asking because setting `informational` at this point will also decrease the level for everything else logged with e.g. `error_logger:info_msg/2` which IMHO is not really desired.